docs/compute/resources_requests_and_limits.md

```diff
@@ -20,18 +20,11 @@ Note: dedicated CPUs (and isolated emulator thread) are ignored here as they hav
 ### Auto CPU limits
 KubeVirt provides two ways to automatically set CPU limits on VM(I)s:
 
-- Enable the `AutoResourceLimitsGate` feature gate.
+- The namespace where the VMI will be created has a ResourceQuota containing cpu limits.
```

Comment:
I would entirely rephrase it to something like:
```
KubeVirt automatically set CPU limits on VM(I)s if all the following conditions are true:

The namespace where the VMI will be created has a ResourceQuota containing cpu limits.
The VMI has no manually set cpu limits.
The VMI is not requesting dedicated CPU.

Additionally, you can add the namespaceLabelSelector in the KubeVirt CR, forcing CPU limits to be set.

In both cases, the VM(I) created will have a CPU limit of 1 per vCPU.
```
WDYT?

Reply:
Works for me, Done.